spec/rspec/scenarios_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+require 'capybara/rspec'
+
+RSpec.configuration.before(:each, { file_path: "./spec/rspec/features_spec.rb" } ) do
+  @in_filtered_hook = true
+end
+
+if RSpec::Core::Version::STRING.to_f >= 3.0 || RUBY_ENGINE == 'rbx'
```

Comment:
@twalpole I don't know if it's good idea to put if statement for `rbx`. I'm unfamiliar with it at all. Do you think it's acceptable?